ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh

```diff
@@ -28,6 +28,8 @@ function mirror_test_images() {
         ssh "${SSHOPTS[@]}" "root@${IP}" bash - << EOF
 openshift-tests images --to-repository ${DEVSCRIPTS_TEST_IMAGE_REPO} > /tmp/mirror
 oc image mirror -f /tmp/mirror --registry-config ${DS_WORKING_DIR}/pull_secret.json
+oc patch configs.samples cluster \
+        --type merge --patch "{\"spec\":{\"samplesRegistry\":\"${DS_REGISTRY_DNS_NAME}\"}}"
```

Comment:
The already defined `DS_REGISTRY` can also be used, since it has already the port specified.
Also, I think it would be be better to put it into a separate function (it's not mirroring a test image, and it may require additional steps)